ghc-mod.cabal

```diff
@@ -201,6 +205,8 @@ Executable ghc-mod
                       , mtl       < 2.3 && >= 2.0
                       , ghc       < 7.11
                       , fclabels  ==2.0.*
+                      , optparse-applicative ==0.11.*
+                      , ansi-wl-pprint ==0.6.*
```

Comment:
ansi-wl-pprint is what optparse-applicative wants, if I'm not mistaken. Might remove the other one though.